packages/kit/scripts/update_version.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+import { resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
+const pathToPackageJson = resolve(__dirname, '..', 'package.json')
+const pathToVersionModule = resolve(__dirname, '..', 'src', 'version.js')
+
+const pkg = JSON.parse(await readFile(pathToPackageJson, { encoding: 'utf-8' }));
```

Comment:
I could have used `import pkg from '../package.json' assert { type: 'json' }` here, but I choose to do not so because:
1. `TypeScript` yelled that it wants the `module` option set to `esnext`
2. `eslint` does not support the syntax because it is experimental (https://github.com/eslint/eslint/discussions/15305)

![image](https://github.com/sveltejs/kit/assets/21335119/12cc9cb7-fe83-46ae-ad2f-eeeff5621e0c)

We can also use the import assertion variant, if we add a `@ts-ignore` comment and add this file to the `excludes` array of `eslint`